tests/LogTargetTest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace yiiunit\extensions\debug;
+
+use yii\debug\LogTarget;
+use yii\debug\Module;
+
+class LogTargetTest extends TestCase
+{
+    public function testGetRequestTime()
+    {
+        $logTarget = new LogTarget(new Module('debug'));
+        $actual = $this->invoke($logTarget, 'getRequestTime');
+        $this->assertTrue(is_float($actual));
```

Comment:
This is debatable of course, however the test should ensure that method actually returns request start time and not some random float.